hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/EventCondition.java

```diff
@@ -100,6 +118,7 @@ public boolean match(Event value) {
     private static final String ID = "id";
     private static final String CTIME = "ctime";
     private static final String TEXT = "text";
+    private static final String CATEGORY = "category";
     private static final String CONTEXT = "context.";
```

Comment:
I'm not sure I like that you can perform matching against context, this sort of breaks our general rule of not performing matching/filtering against data context and may lead to requests for other data to be able to do it and the other types are not set up in a fashion to provide that sort of feature.